requirements-dev.txt

```diff
@@ -16,3 +16,4 @@ pyyaml==5.3.1
 click==7.1.2
 ruamel.yaml==0.16.12
 pytest
+brotli<=1.0.9
```

Comment:
Pipelines were failing in py27: https://github.com/solvebio/solvebio-python/actions/runs/6150337104/job/16688071606

It was due to this breaking change: https://github.com/google/brotli/issues/1074